COORDINATION.md

```diff
@@ -10,9 +10,24 @@ Initial bootup part covers: Server is Powered On, DPU receives power and starts
 
 Coordinated shutdowns, reboots, crashes, error handling will be details on a separate page.
 
+## Terms
+
```

Comment:
we have a table here https://github.com/opiproject/opi-prov-life/blob/main/BOOTSEQ.md#terms